Formula/mrbayes.rb

```diff
@@ -12,31 +12,26 @@ class Mrbayes < Formula
     sha256 "de35643f5dc2c6f2234aed0d121b5446688fda285d0ecdd450a0258afc728717" => :x86_64_linux
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "beagle" => :recommended
-  depends_on "open-mpi" => :recommended
+  depends_on "beagle"   => :recommended
```

Comment:
```suggestion
  depends_on "beagle"
```
Is there a good use case for using MrBayes without `beagle`?